src/timetell/datapunt.sql

```diff
@@ -2495,3 +2354,168 @@ LEFT JOIN
+            c2.fromdate,
+            c2.todate,
+            c2.name AS prj_name,
+            c2.calc_hours,
+            c2.calc_costs,
+            c2.nr AS "Project Nummer"
+           FROM "{schemaname}"."PRJ" a2
+             JOIN "{schemaname}"."PRJ" b2 ON a2.prj_id = b2.parent_id AND a2.parent_id IS NULL
+             JOIN "{schemaname}"."PRJ" c2 ON b2.prj_id = c2.parent_id
+             LEFT JOIN ( SELECT "PRJ_LINK".prj_id,
+                    "PRJ_LINK".emp_id
+                   FROM "{schemaname}"."PRJ_LINK"
+                  WHERE "{schemaname}"."PRJ_LINK".prjleader = 1
+                  GROUP BY "PRJ_LINK".prj_id, "PRJ_LINK".emp_id) prjpl2 ON b2.prj_id = prjpl2.prj_id
+             JOIN "{schemaname}"."EMP" e2 ON e2.emp_id = prjpl2.emp_id
+             LEFT JOIN ( SELECT lp.dim_id AS prj_id,
+                    max(soi.item::text) AS basis_team_name
+                   FROM "{schemaname}"."VW_LABEL_PRJ" lp
+                     LEFT JOIN "{schemaname}"."SYS_OPT_ITM" soi ON lp.item_id = soi.item_id
+                  WHERE to_char(lp.todate, 'YYYYMMDD') = '29991231'
+                  GROUP BY lp.dim_id) vo2 ON vo2.prj_id = b2.prj_id
+             LEFT JOIN "{schemaname}"."CUST" cust2 ON c2.cust_id = cust2.cust_id) project
+     LEFT JOIN ( SELECT h.prj_id,
+            h.date AS hrs_date,
+            h.hours AS hrs_hours,
+            h.internalrate AS hrs_internalrate,
+            h.rate AS hrs_rate,
+            h.hoursrate AS hrs_hoursrate,
+            h.hoursinternalrate AS hrs_hoursinternalrate,
+            emp.name AS emp_name,
+            vafd.name AS org_name,
+            emp.empcat AS emp_empcat,
+            vafd.name::text AS "Medew. Verantw.",
+            act.name AS act_name
+           FROM "{schemaname}"."HRS" h
+             JOIN "{schemaname}"."EMP" emp ON h.emp_id = emp.emp_id
+             JOIN (
+				 select distinct teo.emp_id, top.org_id, top.name from (
+				select org_id, name from "{schemaname}"."ORG"
+				 where todate=to_date('2999-12-31','YYYY-MM-DD') and parent_id is null) top
+				JOIN (select emp_id, org_id from "{schemaname}"."EMP_ORG") teo ON top.org_id=teo.org_id
+				where teo.emp_id not in
+					(Select xeo.emp_id from "{schemaname}"."EMP_ORG" xeo JOIN "{schemaname}"."ORG" xo ON xeo.org_id=xo.org_id
+                              and xo.parent_id is not null
+                              where xo.todate=to_date('2999-12-31', 'YYYY-MM-DD') and xeo.type=0)
+				UNION
+					Select eo.emp_id, o.org_id, o.name from "{schemaname}"."EMP_ORG" eo
+				 		JOIN "{schemaname}"."ORG" o ON eo.org_id=o.org_id
+                        and o.parent_id is not null
+                              where o.todate=to_date('2999-12-31','YYYY-MM-DD') AND eo.type = 0
+                              group by eo.emp_id, o.org_id, o.name
+
+			 ) vafd ON vafd.emp_id = h.emp_id
+             JOIN "{schemaname}"."ACT" act ON h.act_id = act.act_id) uren ON project.prj_prj_id = uren.prj_id;
```

Comment:
Dude... What am I looking at :). A separate file is probably in everyone's best interest next time.